Project5.md

```diff
@@ -84,7 +84,7 @@ You will be adding three new containers to your application:
 
    - ElasticSearch based on the 'elasticsearch:2.0' image on Dockerhub
    - Kafka based on the 'spotify/kafka' image on Dockerhub
-   - The backend search indexer based on my new tp33/django:1.3 image on Dockerhub (kafka-python upgraded)
+   - The backend search indexer based on my new tp33/django:latest image on Dockerhub (kafka-python upgraded)
```

Comment:
If we switch to the new auto-building image on Docker hub the name will change to tp33/django-docker